Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -965,6 +966,8 @@ public struct HTTPClientError: Error, Equatable, CustomStringConvertible {
     public static let readTimeout = HTTPClientError(code: .readTimeout)
     /// Remote connection was closed unexpectedly.
     public static let remoteConnectionClosed = HTTPClientError(code: .remoteConnectionClosed)
+    /// Request failed.
+    public static let failed = HTTPClientError(code: .failed)
```

Comment:
Can we call this `requestFailed` instead of just `failed`?

Reply:
I was aligning it with the naming of `cancelled` which is documented as a cancelled request. What do you prefer?